central/src/gitlab.rs

```diff
@@ -24,6 +24,28 @@ pub struct GitLabProjectAccessTokenProvider {
     client: reqwest::Client,
 }
 
+/// Derive the bridgehead configuration repository from the beam id of the requester.
+/// Example return value: "bridgehead-configurations/bridgehead-config-berlin"
+fn derive_bridgehead_config_repo_from_beam_id(requester: &AppId) -> Result<String, String> {
+    let mut parts = requester.as_ref().splitn(3, '.');
+    let (_app, proxy, broker) = (parts.next().unwrap(), parts.next().unwrap(), parts.next().unwrap());
+
+    let group = match broker {
+        "broker.ccp-it.dktk.dkfz.de" => "bridgehead-configurations", // ccp
+        "broker.bbmri.sample.de" => "bbmri-bridgehead-configs", // bbmri
+        "broker.bbmri.de" => "bbmri-bridgehead-configs", // gbn
+        "test-no-real-data.broker.samply.de" => "bridgehead-configurations", // cce, itcc, kr
+        "broker.hector.dkfz.de" => "dhki", // dhki
+        _ => return Err(format!("Bridgehead configuration repository not known for beam id {requester}")),
+    };
+
+    Ok(match group {
+        "bridgehead-configurations" => format!("{group}/bridgehead-config-{proxy}"),
+        "dhki" => format!("{group}/{proxy}-bk"),
+        _ => format!("{group}/{proxy}"),
+    })
+}
```

Comment:
@torbrenner I implemented the list you sent me; please confirm this is correct. I found one more irregularity: projects in the `bridgehead-configurations` group need the `bridgehead-config-` prefix, whereas projects in the `bbmri-bridgehead-configs` group don't. I can't check the other groups because I don't have permissions to see them on GitLab.